README.md

```diff
@@ -244,10 +240,10 @@ vertex-pipelines-end-to-end-samples
 
 Make sure that you give the ML pipeline a unique name in the `@pipeline` decorator.
 
-To run your pipeline, use `make run` as before:
+To run your pipeline, use `make run` as before (optionally adding parameter to wait until pipeline is finished before returning - defaults to false):
 
 ```bash
-make run pipeline=your_new_pipeline
+make run pipeline=your_new_pipeline [ wait=<true|false> ]
 ```
 
 You will also need to add an E2E test - copy and paste the `training` or `prediction` example in [pipelines/tests/](/pipelines/tests/).
```

Comment:
remove this ref to E2E tests as well (just take out the whole line)

Have a look through the codebase and see where else we refer to E2E tests. The only references that should remain are talking about the CI/CD pipeline `e2e-test.yaml`, not any other code that we have removed

Reply:
I would suggest this change:

`Make sure to update any unit tests and end-to-end tests in line with your changes to the pipelines` -> `Make sure to update any unit tests in line with your changes to the pipelines`

Otherwise I think you can leave it as-is